.github/workflows/make.yml

```diff
@@ -9,15 +9,18 @@ on:
       - main
 
 jobs:
-  run-tests:
-    name: Run tests
+  run-lint-build-tests:
+    name: Run Lint, Build and Tests
     runs-on: ubuntu-22.04
     defaults:
       run:
         working-directory: .
     steps:
       - name: Check out Git repository
         uses: actions/checkout@v3
+      
+      - name: Verify linting
+        run: find ./ -iname *.c -o -iname *.h  -path ./prototype/mnist -prune | xargs clang-format --dry-run --Werror
```

Comment:
Do we allow the PR to be merged even if the Lint fails?

Reply:
Well its going to show check fail